lookup_plugins/selinux_modules.py

```diff
@@ -0,0 +1,50 @@
+# python 3 headers, required if submitting to Ansible
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+        lookup: selinux_modules
+        author: Petr Lautrbach <plautrba@redhat.com>
+        version_added: "0.1"
+        short_description: Read SELinux modules
+        description:
+            - This lookup returns the dict with SELinux modules
+        options:
+          _modules:
+            description: modules to list
+            required: False
+"""
+
+import subprocess
+
+from ansible.plugins.lookup import LookupBase
+from ansible.utils.display import Display
+
+display = Display()
```

Comment:
display is unused?

Reply:
https://github.com/linux-system-roles/selinux/pull/56/commits/1c27466c3373d89950c69e384dfd201882e677cc